src/modules/events/controllers/events.js

```diff
@@ -0,0 +1,64 @@
+const Event = require('../models/Event')
+
+const eventInstance = Event({})
```

Comment:
```suggestion
const event = Event({})
```
ou
```suggestion
const eventModel = Event({})
```
`eventInstance` me parece genérico, talvez deixar `event` pois é um `Event` ou indicar que é um model.

Reply:
com certeza `eventModel`, pois testei com `event` apenas e conflitou com a outra variável event.